index.html

```diff
@@ -436,6 +444,11 @@ <h2>
           {{Document}} |document:Document| are as follows:
         </p>
         <ol class="algorithm">
+          <li>If |document:Document|.{{Document/[[OverridePosture]]}} contains
```

Comment:
As discussed in our call, the internal slot should be associated with a top-level traversable. You also need a way to go from a Document to a top-level traversable.
* A Document has a _relevant global object_, which is a Window in this case.
* A [Window](https://html.spec.whatwg.org/multipage/nav-history-apis.html#window-navigable) gives you a navigable.
* A navigable gives you a top-level traversable.

"document's relevant global object's navigable's top-level traversable" is a mouthful, but it works.

Reply:
`"head explode"`

When you said to look what some HTML item contains, before this explanation I would not found that three level deep path.

- Document: I would have went first to this location https://html.spec.whatwg.org/multipage/dom.html#document. And tried to find something called "top-level traversable" or similar between `partial interface Document {` and `}`. And though I have to find it in there.  After I read "A [Window](https://html.spec.whatwg.org/multipage/nav-history-apis.html#window-navigable) gives you a navigable." I understood that when you said "containing" I would need to look after the `partial interface Document {` stuff. Even now if I look [Document](https://html.spec.whatwg.org/multipage/dom.html#document) page I do not know where you pulled `relevant global object` from that page.
- "A [Window](https://html.spec.whatwg.org/multipage/nav-history-apis.html#window-navigable) gives you a navigable.": **Maybe** after this I remember , when you would give me link to [Window](https://html.spec.whatwg.org/multipage/nav-history-apis.html#window) I would find what it contains. 
- "A navigable gives you a top-level traversable.": Not easy but to conclude this maybe I should go to [navigable](https://html.spec.whatwg.org/multipage/document-sequences.html#navigable) and understand that ... sorry lost... In [navigable](https://html.spec.whatwg.org/multipage/document-sequences.html#navigable) page does this "A parent, a [navigable](https://html.spec.whatwg.org/multipage/document-sequences.html#navigable) or null." line indicate if it is "top-level traversable"? Is there some field in document where the value is "A parent, a [navigable](https://html.spec.whatwg.org/multipage/document-sequences.html#navigable) or null."?


In https://github.com/JuhaVainio/W3C-device-posture/pull/1/commits/cf6fcf35101ce123b86a527595b51c384627c744 I wrote this change

Reply:
> `"head explode"`

That's where reading other specs is really helpful, as these concepts are in different parts of the HTML spec so one of the quickest ways to see what can be done is to see what others have already managed to do (getting a navigable from a document is very common, and you can use the Webdex page I sent you yesterday or even GitHub search for this).

> * Document: I would have went first to this location https://html.spec.whatwg.org/multipage/dom.html#document. And tried to find something called "top-level traversable" or similar between `partial interface Document {` and `}`. And though I have to find it in there.

Yeah, the concepts are in different parts of the spec. https://html.spec.whatwg.org/multipage/document-sequences.html contains several other algorithms related to Documents which are relevant to this discussion.

> After I read "A [Window](https://html.spec.whatwg.org/multipage/nav-history-apis.html#window-navigable) gives you a navigable." I understood that when you said "containing" I would need to look after the `partial interface Document {` stuff. Even now if I look [Document](https://html.spec.whatwg.org/multipage/dom.html#document) page I do not know where you pulled `relevant global object` from that page.

This one comes from https://html.spec.whatwg.org/multipage/webappapis.html#concept-relevant-global. "Global object" is a term that comes from JavaScript -- https://developer.mozilla.org/en-US/docs/Glossary/Global_object. The important thing to know in this case is `Window` is a kind of global object, and since this API is only exposed to the Window context (and not workers), a Document's relevant global object will always be a Window (and not a worker).

Finding all of this out on your own is very time-consuming, but reading what others have done helps **a lot** here (a lot of specs need to refer to "relevant global object").

> * "A navigable gives you a top-level traversable.": Not easy but to conclude this maybe I should go to [navigable](https://html.spec.whatwg.org/multipage/document-sequences.html#navigable) and understand that ... sorry lost... In [navigable](https://html.spec.whatwg.org/multipage/document-sequences.html#navigable) page does this "A parent, a [navigable](https://html.spec.whatwg.org/multipage/document-sequences.html#navigable) or null." line indicate if it is "top-level traversable"? Is there some field in document where the value is "A parent, a [navigable](https://html.spec.whatwg.org/multipage/document-sequences.html#navigable) or null."?

I mentioned this in another comment; the relevant definition here is https://html.spec.whatwg.org/multipage/document-sequences.html#nav-top, which is the definition of "a navigable's top-level traversable".